docs/content/uijson_format/json_objects.rst

```diff
@@ -0,0 +1,313 @@
+   "label": "Select Points or Curve",
+   "value": ""
+   }
+   }
+
+.. figure:: ./images/object_param.png
+
+
+.. _data_parameter:
+
+Geoscience ANALYST Data parameter
+=================================
+
+Creating a parameter to choose a Geoscience ANALYST object's data requires extra members:
+
+dataType ``str``
+   Describes the type of data to filter. One or more of these key words: ``Integer``, ``Float``, ``Text``, ``Referenced``, ``Vector``, ``DataTime``, ``Geometric``, ``Boolean``, or ``Text``
+association ``str``
+   Describes the geometry of the data. One or more of these key words: ``Vertex``, ``Cell``, or ``Face``.
+parent ``str``
+   Either a *UUID* of the parent or the name of the :ref:`Object parameter <object_parameter>` json object to allow the user to choose the mesh.
+isValue ``bool``
+   Describes whether to read the **value** (``float``) or **property** (``str``) member. If not given, the value member is an *UUID* and is considered a *drop-down data parameter*. If this member is given along with **property**, then an icon is added to the UI element, which switches between te **value** (line edit) and **property** (drop-down) choices. This value is updated on export depending on the style choice (``float`` or ``str``)
+property ``str``.
+   Data *UUID*  that is selected when **isValue** is present.  Geoscience ANALYST Pro will update this value on export.
+min ``float``
+    Optional minimum value allowed for validator of the **value** member. The default is the minimum numeric limits of float.
+max ``float``
+    Optional maximum value allowed for validator of the **value** member. The default is the maximum numeric limits of float.
+precision ``int``
+    Optional number of decimal places for the value.
+
+
+Drop-down data parameter
+------------------------
+In this example, the object parameter *data_mesh* is also given for reference.
+
+.. code-block:: json
+
+   {
+   "data_channel": {
+   "main": true,
+   "association": "Vertex",
+   "dataType": "Float",
+   "label": "Data channel",
+   "parent": "data_mesh",
+   "value": ""
+   },
+   "data_mesh": {
+   "main": true,
+    "meshType": ["{202C5DB1-A56D-4004-9CAD-BAAFD8899406}" ,
+      "{6A057FDC-B355-11E3-95BE-FD84A7FFCB88}"],
+   "main": true,
+   "label": "Select Points or Curve",
+   "value": ""
+   }
+   }
+
+
+.. figure:: ./images/data_param.png
+
+
+
+Data or value parameter
+-----------------------
+In some cases, a parameter may take its data from a Geoscience ANALYST object or simply a ``float`` value. The use of the member **isValue** and **property** together allows for the UI to switch between these two cases. In this example, the **property** member will initially be active. The **value** will be set to 1.0 and show up when the icon is clicked. The **uncertainty channel** object depends on the **data_mesh** object. The drop-down selection will filter data from the chosen object that is located on the vertices and is float (second image below).
```

Comment:
```suggestion
In some cases, a parameter may take its data from a Geoscience ANALYST object or simply a ``float`` value. The use of the member **isValue** and **property** together allows for the UI to switch between these two cases. In this example, the **value** member will initially be active. The **value** will be set to 1.0 and show up when the icon is clicked. The **uncertainty channel** object depends on the **data_mesh** object. The drop-down selection will filter data from the chosen object that is located on the vertices and is float (second image below).
```

Reply:
Actually, by changing the isValue and then mentioning the top image, I think my change is consistent now.